deploy.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -ev
+echo `zip -r extension.zip extension/`
+echo `grunt`
```

Comment:
What I mean is - there is no need for the script to be an external file. I've successfully used Travis CI script deployment with an inline command. Just like other commands you can pass to `.travis.yml` directives`.

And even if we do use a separate file for the deploy scripts you can just write it like that:

``` bash
#!/bin/bash
set -ev
zip -r extension.zip extension/
grunt
```

It will do the same thing as far as I know and I'm curious why you need the `echo` + backticks approach. I know what I does, but I think the end result is more or less the same.